client/src/index.jsx

```diff
@@ -1,45 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
+import axios from 'axios';
+
 import Trial from './components/trial.jsx';
 import Header from './components/header.jsx';
-import axios from 'axios';
+import Inventory from './components/inventory.jsx';
 
 class App extends React.Component {
   constructor (props) {
     super (props);
     this.state = {
       titleAndAuthor: {},
```

Comment:
is there a reason to have these two properties together in an object? It seems like it might be better to just have a title and an author property on this state